src/modules/item/selectors.ts

```diff
@@ -25,9 +25,9 @@ export const getWalletItems = createSelector<RootState, Item[], string | undefin
 )
 
 export const getAuthorizedItems = createSelector<RootState, Collection[], Item[], string | undefined, Item[]>(
-  getAuthorizedCollections,
-  getItems,
-  getAddress,
+  state => getAuthorizedCollections(state),
```

Comment:
I'm curious about why it was changed to this

Reply:
This was due to circular deps between the selectors. Passing `getAuthorizedCollections` as reference there would be undefined, but calling it within that lambda works fine on runtime.